python/lsst/summit/utils/utils.py

```diff
@@ -820,3 +823,90 @@ def getFilterSeeingCorrection(filterName):
             return (762./500.)**0.2
         case _:
             raise ValueError(f"Unknown filter name: {filterName}")
+
+
+def getCdf(data, scale):
+    """Return an approximate cumulative distribution function scaled to
+    the [0, scale] range.
+
+    Parameters
+    ----------
+    data : `np.array`
+        The input data.
+    scale : `int`
+        The scaling range of the output.
+
+    Returns
+    -------
+    cdf : `np.array` of `int`
+        A monotonically increasing sequence that represents a scaled
+        cumulative distribution function, starting with the value at
+        minVal, then at (minVal + 1), and so on.
+    minVal : `float`
+        An integer smaller than the minimum value in the input data.
+    maxVal : `float`
+        An integer larger than the maximum value in the input data.
+    """
+    flatData = data.ravel()
+    size = flatData.size - np.count_nonzero(np.isnan(flatData))
+
+    minVal = np.floor(np.nanmin(flatData)) - 1.0
+    maxVal = np.ceil(np.nanmax(flatData)) + 1.0
```

Comment:
Given you're using `floor` and `ceil` here, do we need the -1 and +1 on top of that? Shouldn't those be enough to guarantee covering the whole range?

Reply:
About adjusting the `maxVal`, the as-is `digitizeData` [implementation](https://github.com/lsst-sitcom/summit_utils/blob/f5620ec3fc723dbae77b9af24a9387d139838538/python/lsst/summit/utils/utils.py#L911) would fail in the case that the largest data point is an integer and falls on rightmost edge, because it'd then want the non-existing `(maxVal - minVal)` bin. Making `maxVal` a little larger does the trick. 

Not adjusting the `minVal` would work too, I think.

Reply:
Ah, I see, that makes sense, thanks.